chaoscenter/web/src/api/core/probe/getProbesInExperimentRun.ts

```diff
@@ -0,0 +1,125 @@
+import { gql, useLazyQuery } from '@apollo/client';
+import type { GqlAPILazyQueryRequest, GqlAPILazyQueryResponse } from '@api/types';
+import type { Mode, Probe, Status } from '@api/entities';
+
+export interface GetProbesInExperimentRunRequest {
+  projectID: string;
+  experimentRunID: string;
+  faultName: string;
+}
+
+export interface ProbeInRuns {
+  probe: Probe;
+  mode: Mode;
+  status: Status;
+}
+
+export interface GetProbesInExperimentRunResponse {
+  getProbesInExperimentRun: ProbeInRuns[];
+}
+
+export function getProbesInExperimentRun({
+  ...options
+}: GqlAPILazyQueryRequest<GetProbesInExperimentRunResponse, GetProbesInExperimentRunRequest>): GqlAPILazyQueryResponse<
+  GetProbesInExperimentRunResponse,
+  GetProbesInExperimentRunRequest
+> {
+  // Query to List workflows
+  const [getProbesInExperimentRunQuery, result] = useLazyQuery<
+    GetProbesInExperimentRunResponse,
+    GetProbesInExperimentRunRequest
+  >(
+    gql`
+      query GetProbesInExperimentRun($projectID: ID!, $experimentRunID: String!, $faultName: String!) {
+        getProbesInExperimentRun(projectID: $projectID, experimentRunID: $experimentRunID, faultName: $faultName) {
+          probe {
+            name
+            type
+            infrastructureType
+            kubernetesHTTPProperties {
```

Comment:
Please add `attempt` and `evaluationTimeout` for all probes, only CMD probe has `attempt` in this implementation